resources/views/layout.blade.php

```diff
@@ -13,6 +13,9 @@
 <link href='{!! asset("/css/wedding.css")!!}' rel='stylesheet'>
 <link href='{!! asset("/css/banner.css")!!}' rel='stylesheet'>
 <link href='{!! asset("/css/animate.css")!!}' rel='stylesheet'>
+
+<!--JS -->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
Why was jquery 3.2.1 added here? Is it necessary? A local version is provisioned in the home.blade.php. Perhaps move that to the layout?